Index: GameIzo/src/GameLogic/Spells/SpellEffects/TeleportationEffect.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- GameIzo/src/GameLogic/Spells/SpellEffects/TeleportationEffect.java	(date 1553022920000)
+++ GameIzo/src/GameLogic/Spells/SpellEffects/TeleportationEffect.java	(date 1553022968288)
@@ -20,12 +20,16 @@
         switch(champion.getDirection()) {
             case UP:
                 champion.setY(old_y - delta);
+                break;
             case DOWN:
                 champion.setY(old_y + delta);
+                break;
             case RIGHT:
                 champion.setX(old_x + delta);
+                break;
             case LEFT:
                 champion.setX(old_x - delta);
+                break;
         }
     }
 
